.github/workflows/figma-sync.yml

```diff
@@ -0,0 +1,215 @@
+name: Design Token Sync
+
+on:
+  # Manual execution possible
+  workflow_dispatch:
+    inputs:
+      force_sync:
+        description: 'Force sync even if no changes'
+        required: false
+        default: 'false'
+        type: boolean
+  # Temporal trigger for testing
+  pull_request:
+    branches:
+      - develop
```

Comment:
This part is temporal testing approach. Before we merge this, we need to remove this.